src/main/java/frc/robot/subsystems/drive/SwerveSubsystem.java

```diff
@@ -230,4 +220,15 @@ public Command followTrajectoryCommand(PathPlannerTrajectory traj, boolean isFir
       })
   );
 }
+public Command followTrajectoriesCommand(ArrayList<PathPlannerTrajectory> trajectories, boolean stopWhenDone) {
+  // in your code that will be used by all path following commands.
+  SequentialCommandGroup fullCommand = new SequentialCommandGroup(new InstantCommand(() -> {
+    // Reset odometry for the first path you run during auto
+        // this.setPose(trajectories.get(0).getInitialHolonomicPose());
```

Comment:
1) why is this commented out? wouldn't we want this? if it was temporary consider making it a function argument or just removing it
2) maybe we should just straight up remove commented out code?

Reply:
yea just remove it lol